__tests__/routes/newsfeed.test.js

```diff
@@ -47,7 +47,7 @@ describe('newsfeeds router endpoints', () => {
   });
 
   describe('POST /newsfeed', () => {
-    it('Should return 200 when newsfeed is added', async () => {
```

Comment:
great catch using the correct status number! remembering the 201 when posting something and 200 when calling something is great attention to detail